### PR TITLE
Including Urban Dictionary

### PR DESCRIPTION
**Pull request by saikartheekb**
Since many people search for the latest lingo or the buzz word or that obscure term that no one really know the definition for and can't seem to find it in a dictionary

### CODE REVIEW

File: Code/Searchor/__init__.py [hunk 1]
```diff
@@ -98,3 +98,8 @@ def SearchQuora(query):
   query = query.replace(" ", "%20")
   query_URL = "https://www.quora.com/search?q=" + query
   return query_URL
+
+def SearchUrbanDictionary(query):
+  query = query.replace(" ", "+")
```
Comment (ArjunSharda) [inline Code/Searchor/__init__.py:103]: Isn't it supposed to be %20 instead of + ?

**Review by ArjunSharda — CHANGES_REQUESTED** [review]
Thanks for the PR and your contribution a lot! I've requested a change. Looks good otherwise for v1.4.0

File: Code/Searchor/__init__.py (hunk 1 above)
Comment (saikartheekb) [reply Code/Searchor/__init__.py:103]: My bad! I will do that.

**Review by ArjunSharda — APPROVED** [review]
Thanks for your contributions! Will be added in v1.4.0

**Comment by ArjunSharda** [comment]
My apologies, I forgot to change the base branch for another PR. I will be committing to a new branch and adding all the code that has to be added.

**Pull request merged**